src/main/java/com/smartbear/ready/jenkins/JenkinsSoapUIProTestRunner.java

```diff
@@ -90,8 +90,8 @@ public void setTestCaseTags(String tags) {
         this.testCaseTags = tags;
     }
 
-    public String getProjectPassword() {
-        return Secret.toString(projectPassword);
+    public Secret getProjectPassword() {
+        return projectPassword;
```

Comment:
Is this method called by Jenkins?

Reply:
I think, yes, because changing the signature affects value in the settings form of a job.

Reply:
Is value of password correctly applying to encrypted project ?

Reply:
I've tested it on my local old version of Jenkins 2.204.5. In the <input value=""> the password is encrypted, but it's changeable correctly and the right password it's applied correctly.